crates/ruma-macros/src/events/event_enum.rs

```diff
@@ -280,10 +280,7 @@ fn expand_into_full_event(
         #[automatically_derived]
         impl #ident {
             /// Convert this sync event into a full event (one with a `room_id` field).
-            pub fn into_full_event(
-                self,
-                room_id: ::std::boxed::Box<#ruma_common::RoomId>,
-            ) -> #full {
+            pub fn into_full_event(self, room_id: #ruma_common::OwnedRoomId) -> #full {
```

Comment:
Why not change this to `AsRef<RoomId>`?

Reply:
I don't see why.

Reply:
Ah nvm, the full event takes ownership of the `room_id` for a field